starknet_devnet/util.py

```diff
@@ -241,3 +242,21 @@ def generate_state_update(previous_state: CarriedState, current_state: CarriedSt
             "storage_diffs": storage_diffs
         }
     }
+
+class JsonErrorHandler:
```

Comment:
Could we leverage use of [Flask.errorhandler](https://flask.palletsprojects.com/en/2.1.x/api/#flask.Flask.errorhandler)? We could catch either status or `StarknetDevnetException` and return JSON there?

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/81/commits/8ab904c4af7208bf5e4441facd4861f5d74ab257